src/components/shared/__tests__/Modal.test.js

```diff
@@ -0,0 +1,32 @@
+import Modal from '../Modal';
+
+const defaultProps = {
+  handleCancelClick: jest.fn(),
+  handleConfirmClick: jest.fn(),
+};
+
+const setup = buildSetup(Modal, defaultProps);
+
+describe('<Modal />', () => {
+  it('should render without crash', () => {
+    const { wrapper } = setup();
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should call handleCancelClick after click on cancel button', () => {
+    const { wrapper } = setup();
+    wrapper
+      .find('button')
+      .at(0)
```

Comment:
You can also find elements by class or text inside. This would make the find more explicit, and less brittle to markup changes.